src/routes/safe/store/actions/transactions/utils/transactionHelpers.ts

```diff
@@ -317,13 +317,13 @@ export type TxToMock = TxArgs & {
   value: string
 }
 
-export const mockTransaction = (tx: TxToMock, safeAddress: string, state: AppReduxState): Promise<Transaction> => {
+export const getMockTransactionServiceModel = (): TxServiceModel => {
```

Comment:
this method does nothing but returns an object with empty properties..

Reply:
yes, that's to mock a TransactionService, if we don't have that generic mock, we need to create the complete object structure to be aligned with the type. then you can use object spread to override the attributes you need.